Libraries/Farmhand/API/Crops/Crop.cs

```diff
@@ -25,8 +23,7 @@ public static void RegisterCrop<T>(CropInformation crop)
                 throw new Exception("objectInformation is null! This likely occurs if you try to register a crop before AfterContentLoaded");
             }
 
-            crop.Id = NextAvailableIndex;
-            NextAvailableIndex++;
+            crop.Id = StardewValley.Game1.content.Load<Dictionary<int, string>>("Data\\Crops").Count()+1;
```

Comment:
It feels like there might be a better way of doing that, but it'll work for now! :)